CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+Changelog
+===
+
+# Initial release
```

Comment:
1.0.0-alpha-1. "Initial release" was supposed to be in the text, but features are fine